### ISSUE TITLE
Wrong namespace for `Duration` message from `std_msgs`

### ISSUE THREAD

**Issue by Th-Havy** [Unity-Technologies/ROS-TCP-Connector#83]
Hi,

The `Duration` message from [std_msgs](http://wiki.ros.org/std_msgs) is in the wrong namespace in this package.

In the file `MessageGeneration>std_msgs>Duration.cs`:
```csharp
// This should be "namespace RosMessageTypes.Std" like all the other standard messages
namespace Unity.Robotics.ROSTCPConnector.MessageGeneration
{
    public class Duration : Message
    {
```

**Comment by peifeng-unity** [comment]
Hi @Th-Havy, thank you for bringing this to our attention. I have created a ticket for the team and we will take a look at it soon.

**Comment by mrpropellers** [comment]
Looks like this was addressed by #91, closing this issue.
